transcom/mymove
# INT-B-18545 safety queue and labels

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A897667&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

> [!CAUTION]
> This work requires 18546.

Added privileges check for SAFETY moves in queues and the SAFETY labels on pages.

### How to test

1. Create a SAFETY move WITH SHIPMENTS
2. Create an office users without SAFETY privileges
3. Check move is NOT in queues
4. Add SAFETY privileges to office users
5. Check move is visible in queues
6. Ensure SAFETY label is visible in all move pages like BLUEBARK/WOUNDED WARRIOR labels
